app/src/main/java/ch/epfl/sweng/runpharaa/notification/FireMessage.java

```diff
@@ -0,0 +1,84 @@
+package ch.epfl.sweng.runpharaa.notification;
+
+import android.util.Log;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.io.OutputStreamWriter;
+import java.net.HttpURLConnection;
+import java.net.URL;
+
+public class FireMessage {
+    //Server key found in firebase
+    private final String SERVER_KEY = "AAAArozt-xs:APA91bEN2m2cVZVUlJ6GMRgacY42W7KF9f3Orpm1bcEKF0WbugfgDfwcKLZ1qmgs4d4EmD7iCgkB3qH6oXfjEk0vQEg2oodWioeYzve__QeCJr2PPgLI4qmHi71L5_EuxL87y1TJNlpX";
```

Comment:
We should think about something to avoid having the API keys in the code

Reply:
Agreed